samples/bluetooth/hci_ipc/nrf5340_cpunet_bis-bt_ll_sw_split.conf

```diff
@@ -24,11 +21,15 @@ CONFIG_BT_BUF_ACL_RX_SIZE=255
 CONFIG_BT_BUF_ACL_TX_SIZE=251
 CONFIG_BT_BUF_CMD_TX_SIZE=255
 
-# Host features
+# Host and Controller common dependencies
+CONFIG_BT_OBSERVER=y
```

Comment:
i wanted to keep broadcaster, observer, central and peripheral all listed at files (y/n), so that users could flip them as required without doing a complete rebuild.

But looks like I missed adding the broadcaster here.